.env-sample

```diff
@@ -1,4 +1,15 @@
 PORT=3000
 DATABASE_URL_DEV=postgresql://username:password@host:localhost:5432/database_name
 DATABASE_URL_TEST=postgresql://username:password@host:localhost:5432/database_name
-DATABASE_URL=postgresql://username:password@host:localhost:5432/database_name
```

Comment:
Why did you remove the third database connection url `DATABASE_URL=postgresql://username:password@host:localhost:5432/database_name` format?